Documentation/Books/Manual/Installation/MacOSX.md

```diff
@@ -98,3 +102,9 @@ showing you the log-file.
 Note that it is possible to install both, the homebrew version and the command-line
 app. You should, however, edit the configuration files of one version and change
 the port used.
+
+Installing using the archive
+----------------------------
+
+Starting from 3.4.0 a _tar.gz_ package is also available for Mac. To install ArangoDB
+using the _tar.gz_ archive, just extract it.
```

Comment:
I'm an favor of:
```suggestion
using the `tar.gz` archive, just extract it.
```
because it's about a file extension. File extensions, file names, paths, startup options, function signatures and small code snippets should use inline code tags (single backticks) IMO.

For names (products, brands, but also if you write something like *is referred to as ...* or *so called ...*) I like the italicized formatting (asterisks).